reconcile/dashdotdb_cso.py

```diff
@@ -47,7 +47,7 @@ def _post(self, manifest):
 
         for item in imagemanifestvuln['items']:
             endpoint = (f'{self.dashdotdb_url}/api/v1/'
-                        f'imagemanifestvuln/{cluster}')
+                        f'ImageManifestVuln/{cluster}')
```

Comment:
@rrati can you review this one? It seems to talk to dashdotdb and not to the cluster

Reply:
I think this change is incorrect.  It doesn't get executed in the dry-run case, so setting back to original code.